var/spack/repos/builtin/packages/r-rcpp/package.py

```diff
@@ -42,4 +42,4 @@ class RRcpp(RPackage):
     version('0.12.6', sha256='1bb54e03b817a3d6ab5917f1bbf5250c6b33f61e466628a378022ed65a010141')
     version('0.12.5', sha256='13449481c91b5271b34d81f462864864c1905bb05021781eee11c36fdafa80ef')
 
-    depends_on('r@3.0.0:', type=('build', 'run'))
+    depends_on('r@3.0.0:', when='@:1.0.4.6', type=('build', 'run'))
```

Comment:
Even though an R version is not listed as a dependency in the DESCRIPTION file I think it is best to leave the original constraint in place.

Reply:
I was not sure about it either, so that is fine by me, too.